packages/block-editor/src/components/index.js

```diff
@@ -15,7 +15,12 @@ export { default as BlockEdit, useBlockEditContext } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
 export { default as BlockIcon } from './block-icon';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
-export { default as __experimentalBlockNavigationList } from './block-navigation/list';
+export {
+	default as __experimentalBlockNavigationList,
+	BlockNavigationContext as __experimentalBlockNavigationContext,
+	ListItemFill as __experimentalBlockNavigationListItemFill,
```

Comment:
It's verbose, but I think this should probably be `BlockNavigationListItemFill` internally as well for consistency.